inference-engine/src/mkldnn_plugin/nodes/mkldnn_normalize_node.cpp

```diff
@@ -164,6 +164,12 @@ struct jit_uni_normalize_kernel_f32 : public jit_uni_normalize_kernel, public ji
             }
         }
 
+        if (!mayiuse(avx512_core_bf16) && mayiuse(avx512_core)) {
+            bf16_emu_ = new bf16_emulation_t<isa>(this, bf16_emu_reserv_1, bf16_emu_reserv_2,
+                bf16_emu_reserv_3, bf16_emu_scratch, bf16_emu_reserv_4);
+            bf16_emu_->init_vcvtneps2bf16();
+        }
+
         this->preamble();
```

Comment:
It seems to me that calling init_vcvtneps2bf16 would be more safe after this->preamble() since init_vcvtneps2bf16 may change some general purpose registers values.

Reply:
Done